draft-qproto-spec.md

```diff
@@ -1,965 +1,965 @@
-|------:|:---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
-|   0x1 | Generic error.                                                                                                                                                                                                                                                                   |
-|   0x2 | Unsupported data. May be sent after the sender sends an [init packet](#init-packets) to indicate that the receiver does not support this codec. The sender MAY send another packet of this type with the same `stream_id` to attempt reinitialization with different parameters. |
-
-Feedback
---------
-The following packet MAY be sent from the receiver to the sender.
-
-| Data                | Name               | Fixed value  | Description                                                                                                                                                                                                                                         |
-|:--------------------|:-------------------|-------------:|:----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
-| b(16)               | `stats_descriptor` |       0x8002 | Indicates this is a statistics packet.                                                                                                                                                                                                              |
-| b(16)               | `stream_id`        |              | Indicates the stream ID for which this packet is relevant to. MAY be 0xffff to indicate all streams.                                                                                                                                                |
-| b(64)               | `bandwidth`        |              | Hint that indicates the available receiver bandwith, in bits per second. MAY be 0, in which case *infinite* MUST be assumed. Senders SHOULD respect it. The figure should include all headers and associated overhead.                              |
-| u(32)               | `fec_corrections`  |              | A counter that indicates the total amount of repaired packets (packets with errors that FEC was able to correct). If FEC is not used, MAY be zero.                                                                                                  |
```

Comment:
somehow the "MAY be zero" seems weird. It's just: `<size>` if FEC used, or `0` otherwise. The "MAY" seems to imply a choice that does not exist.